src/Components/OATGraphViewer/Internal/OATGraphCustomNode.tsx

```diff
@@ -136,7 +159,13 @@ const OATGraphCustomNode: React.FC<IOATGraphCustomNodeProps> = ({
     const onIdBlur = () => {
         setIdEditor(false);
         const prevId = data.id;
-        if (data.id !== idText) {
+        // Only apply change if value is error free
+        if (
+            data.id !== idText &&
```

Comment:
Do you know if IDs are case sensitive? If not, then we might need to do a case insensitive comparison here.

Reply:
@jamescarpinter [DTMI](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier) mentions `Equivalence of DTMIs is case-sensitive.`